src/routes/+page.server.ts

```diff
@@ -12,18 +14,53 @@ export const load = async () => {
 };
 
 export const actions: Actions = {
-  submitReport: async ({ request }) => {
-    const formData = await request.formData();
-    const isValid = validateForm(formData);
+  submitReport: handleContactForm(
+    async ({
+      description,
+      title,
+      teamId,
+      author,
+      priority,
+      attachments,
+      type,
+      steps,
+      technical
+    }) => {
+      const priorityLabel: Record<string, string> = {
+        low: '🟢  **Low**',
+        medium: '🟡  **Medium**',
+        high: '🟠  **High**',
+        critical: '🔴  **Critical**'
+      };
```

Comment:
Doesn't these values are in other palces?
I recommend add a dictionary somewhere to store these values